release_notes_generator/action_inputs.py

```diff
@@ -123,16 +131,12 @@ def validate_inputs():
         if not isinstance(chapters_to_pr_without_issue, bool):
             raise ValueError("Chapters to PR without issue must be a boolean.")
 
-        verbose = ActionInputs.get_verbose()
-        if not isinstance(verbose, bool):
-            raise ValueError("Verbose logging must be a boolean.")
-
         logging.debug(f'Repository: {owner}/{repo_name}')
         logging.debug(f'Tag name: {tag_name}')
         logging.debug(f'Chapters JSON: {chapters_json}')
-        logging.debug(f'Warnings: {warnings}')
         logging.debug(f'Published at: {published_at}')
         logging.debug(f'Skip release notes label: {skip_release_notes_label}')
+        logging.debug(f'Verbose logging: {verbose}')
```

Comment:
Not a best practice to be using f strings for logging. Probably will be solved later in pylint / black formating issue.

Reply:
Yeah, I would like to solve it in Issue with pylint and black when we create list of rules to follow.